src/Command/CodeStudio/CodeStudioCiCdVariables.php

```diff
@@ -10,13 +10,64 @@ class CodeStudioCiCdVariables {
    * @return array<mixed>
    */
   public static function getList(): array {
-    return array_column(self::getDefaults(), 'key');
+    //getlist is being utilised in pipeline-migrate command. By default command is supporting drupal project but going forward need to support both drupal and nodejs project.
+    return array_column(self::getDefaultsForPhp(), 'key');
   }
 
   /**
-   * @return array<mixed>
-   */
-  public static function getDefaults(?string $cloudApplicationUuid = NULL, ?string $cloudKey = NULL, ?string $cloudSecret = NULL, ?string $projectAccessTokenName = NULL, ?string $projectAccessToken = NULL, ?string $phpVersion = NULL): array {
+  * @return array<mixed>
+  */
+  public static function getDefaultsForNode(?string $cloudApplicationUuid = NULL, ?string $cloudKey = NULL, ?string $cloudSecret = NULL, ?string $projectAccessTokenName = NULL, ?string $projectAccessToken = NULL, ?string $nodeVersion = NULL): array {
```

Comment:
Isn't this basically the same as getMockGitlabVariables (with my additions)? Seems like they could be combined.

Reply:
It is almost same but there is change of one last variable. Hence we have introduce `getDefaultsForNode` and `getDefaultsForPhp` removing `getDefaults `